### PR TITLE
Added initial membrane potential options to microcircuit example

### PR DESCRIPTION
**Pull request by HanjiaJiang**
To avoid the transient firing burst and fluctuation in the first 10 ms of simulation, options of optimized initial membrane potentials are added for both sli and pynest microcircuit example code. The optimized membrane potential distributions were measured in the stationary state (100th to 200th ms) for all neuron populations in the model. 
In the network parameter file, the user can choose either the original initial values from the previous version, or the optimized initial values. A warning message is printed in the terminal or the log if the user misspells one of the options.

### CODE REVIEW

File: pynest/examples/Potjans_2014/network.py [hunk 1]
```diff
@@ -199,26 +211,32 @@ def create_populations(self):
                     'I_e': self.DC_amp_e[i]
                     }
                 )
+            if self.net_dict['V0_type'] == 'optimized':
+                nest.SetStatus(population, 'V_m', np.random.normal(
```
Comment (AlexVanMeegen) [inline pynest/examples/Potjans_2014/network.py]: This breaks the correct seeding of random number generators in a distributed simulation, see https://nest-simulator.org/random_numbers/
I'd suggest to copy the loop over the threads (line 223) here and replace `['V0_mean']['original']` by `['V0_mean']['optimized'][i]` and the same for `V0_sd`.

File: examples/nest/Potjans_2014/network_params.sli [hunk 2]
```diff
@@ -101,8 +101,39 @@ def
 /neuron_model /iaf_psc_exp def  % neuron model. For PSP-to-PSC conversion to
                                 % be correct, synapses should be current-based
                                 % with an exponential time course
-/Vm0_mean -58.0 def             % mean of initial membrane potential (mV)
-/Vm0_std 10.0 def               % std of initial membrane potential (mV)
+
+% Initial conditions for the membrane potential, options are:
+% (original): uniform mean and std for all populations
+% (optimized): population-specific mean and std from stationary state
```
Comment (AlexVanMeegen) [inline examples/nest/Potjans_2014/network_params.sli]: See my comment below for `network_params.py` :)

File: pynest/examples/Potjans_2014/network_params.py [hunk 3]
```diff
@@ -195,12 +195,21 @@ def get_std_PSP_matrix(PSP_rel, number_of_pop):
     # Relative standard deviation of the delay of excitatory and
     # inhibitory connections (in relative units).
     'rel_std_delay': 0.5,
+    # Initial conditions for the membrane potential, options are:
+    # 'original': uniform mean and std for all populations
+    # 'optimized': population-specific mean and std from stationary state
```
[the remark this answers is not in the document]
Comment (AlexVanMeegen) [reply pynest/examples/Potjans_2014/network_params.py]: Maybe also include a comment what `optimized` means here, i.e. a reduction of the initial burst of network activity.

[the remark this answers is not in the document]
Comment (HanjiaJiang) [reply pynest/examples/Potjans_2014/network_params.py]: Hi, thanks for reviewing this!
Would the description be better like this?
`
    # 'optimized': population-specific mean and std, allowing a reduction of
    # the initial activity burst in the network
`

[the remark this answers is not in the document]
Comment (HanjiaJiang) [reply pynest/examples/Potjans_2014/network_params.py]: Hi, thanks a lot for reviewing this!
Would the description be better like this?
`# 'optimized': population-specific mean and std, allowing a reduction of`
`# the initial activity burst in the network`

**Comment by stinebuu** [comment]
@HanjiaJiang I think something went wrong when you merged master into your branch, could you try to revert to your previous stage so there aren't so many files changed in this PR?

**Comment by HanjiaJiang** [comment]
> @HanjiaJiang I think something went wrong when you merged master into your branch, could you try to revert to your previous stage so there aren't so many files changed in this PR?

Hi, yes I already found there's something wrong in it. I will try to revert it. Thanks.

**Comment by HanjiaJiang** [comment]
> @HanjiaJiang I think something went wrong when you merged master into your branch, could you try to revert to your previous stage so there aren't so many files changed in this PR?

Hi! I have reverted and merged, and pushed again. I suppose my feature branch should be OK now. Please check again. Thanks!

**Comment by HanjiaJiang** [comment]
The git history of this pull request is wrong because of a corrupted rebase. We close it and open a new pull request of the same content #1260.

**Pull request closed without merging**